mujoco_py/utils.py

```diff
@@ -78,12 +83,8 @@ def discover_mujoco():
     - mujoco_path (str): Path to MuJoCo 2.0 directory.
     - key_path (str): Path to the MuJoCo license key.
     """
-    key_path = os.getenv('MUJOCO_PY_MJKEY_PATH')
-    if not key_path:
-        key_path = join(expanduser('~'), '.mujoco', 'mjkey.txt')
-    mujoco_path = os.getenv('MUJOCO_PY_MUJOCO_PATH')
-    if not mujoco_path:
-        mujoco_path = join(expanduser('~'), '.mujoco', 'mujoco200')
+    key_path = join(expanduser('~'), '.mujoco', 'mjkey.txt')
+    mujoco_path = join(expanduser('~'), '.mujoco', 'mujoco200')
```

Comment:
Would it be possible to also check the following paths here: `~/.mujoco/mujoco_{linux,macos,win64}`. By default the zipfiles do get unzipped to those paths, and for example if using [dm_control](https://github.com/deepmind/dm_control) side by side with mujoco-py, currently some unnecessary symlinking needs to be done.